src/__tests__/createMessage.test.ts

```diff
@@ -0,0 +1,42 @@
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+import * as apiClient from '../';
+import { messages, pickMessage, invalidPickMessages } from './fixtures/messages';
+
+const mock = new MockAdapter(axios);
+
+afterEach(() => {
+	mock.reset();
+});
+
+const apiClientInit = new apiClient.APIClient({ apiBase: 'unics-social' });
+
+test('createMessage(): creates message correctly', async () => {
+	mock.onPost(`unics-social/channels/1/messages`, { content: pickMessage.content }).reply(201, {
+		status: 201,
+		error: '',
```

Comment:
Status can be removed from all response bodies (it is specific to `hs_auth` not KB), and error can be removed for responses that are successful (e.g. this one!)